### PR TITLE
feat: "See Plans" CTA for demo room

### PR DESCRIPTION
**Pull request by nikk15**
Demo room CTA to redirect to [https://hubs.mozilla.com/#subscribe](https://hubs.mozilla.com/#subscribe)
As per [designs](https://www.figma.com/file/ReNfwCp2i8NHplHQCMasJf/DemoRoom-MozillasHub?type=design&node-id=1-481&mode=design&t=KYYAalMlXBFWJx3w-0).

Screenshot 2023-07-05 at 3 04 07 PM
Screenshot 2023-07-05 at 3 04 24 PM

### CODE REVIEW

**Comment by nikk15** [comment]
NOTE: This component has very specific styling and composition, hence the new component rather than the existing Lilypad button. If we decide to use this styling further, I can create a BrandButton component specific for this use in Lilypad.

File: src/react-components/layout/Toolbar.scss [hunk 1]
```diff
@@ -17,16 +18,25 @@
...
 
   @media (min-width: theme.$breakpoint-lg) {
     flex: 0;
+  }
+}
 
-    & > * {
-      margin-right: 24px;
-    }
+:local(.centerContent) {
```
Comment (nickgrato) [inline src/react-components/layout/Toolbar.scss:29]: if you can somehow get rid of the camelCase here that would be awesome. If it's baked in, no big.

File: src/react-components/room/MoreMenuPopover.js [hunk 2]
```diff
@@ -19,12 +27,12 @@ function MoreMenuItem({ item, closePopover }) {
           target={item.target || "_blank"}
           rel="noopener noreferrer"
         >
-          <Icon />
+          {item.icon?.src ? <img src={item.icon.src} alt={imageAltText} /> : <Icon />}
```
Comment (nickgrato) [inline src/react-components/room/MoreMenuPopover.js:30]: can you move this logic to line 11? Then you could leave these as <Icon/> and on line 11 it gets assigned the right asset?

File: src/react-components/room/components/SeePlansCTA/SeePlansCTA.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  text-align: center;
+  text-decoration: none;
+  cursor: pointer;
+  color: var(--color-interaction-primary);
+  width: fit-content;
+  min-width: 132px;
+  font-size: 14px;
```
Comment (nickgrato) [inline src/react-components/room/components/SeePlansCTA/SeePlansCTA.scss:18]: are all the font sixes "px" in the project?

**Pull request merged**